File: BilisPera/BilisPera/Categories/UIStackView+Extension.m
//
//  UIStackView+Extension.m
//  BilisPera
//
//  Created by BHJ on 2025/8/12.
//

#import "UIStackView+Extension.h"

@implementation UIStackView (Extension)

-(void)removeAllSubViews{
    for (UIView *subView in self.subviews) {
        [subView removeFromSuperview];
    }
}

@end

File: SkyLoan/SkyLoan_phillipines/Extension/UIStackView+Extension.swift
//
//  UIStackView+Extension.swift
//  SkyLoan_phillipines
//
//  Created by BHJ on 2025/5/14.
//

import UIKit
extension UIStackView{
    
    func addSubViews(views: [UIView]){
        views.forEach { view in
            addArrangedSubview(view)
        }
    }
    
    func removeAllSubViews(){
        subviews.forEach { view in
            removeArrangedSubview(view)
        }
    }
}
